commit 0d27c3e494ed01c8090a83423434f1f48ae142d5
Author: Jelmer VernooÄ³ <jelmer@jelmer.uk>
Date:   Fri Dec 30 11:59:51 2016 +0000

    Avoid asm/socket.h, which is not available on Hurd/BSD.

diff --git a/bot.c b/bot.c
index c47d588..3bd3026 100644
--- a/bot.c
+++ b/bot.c
@@ -15,7 +15,7 @@
    License along with QuasselC.  If not, see <http://www.gnu.org/licenses/>.
  */
 
-#include <asm/socket.h>
+#include <sys/socket.h>
 #include <sys/types.h>
 #include <sys/socket.h>
 #include <netdb.h>
